kong/plugins/statsd/schema.lua

```diff
@@ -11,6 +11,7 @@ local metrics = {
 
 return {
   fields = {
+    prefix = {type = "string", default = "kong."},
```

Comment:
we will need a migration for this value, otherwise existing installations will have a nil prefix. alternatively, we could define the statsd prefix as `conf.prefix or "kong."`, though for clarify i think we would prefer the schema prefix having the default value, and a migration put in place.